version 4/00 intro/05 jquery/README.md

```diff
@@ -5,29 +5,30 @@ basic pillars of web development, consuming third party libraries.
 
 In this demo we will install a legacy library (jquery) via npm, define it as global, and use it. Finally we will end up creating a separate bundle for libraries.
 
-We will start from sample _00 Intro/03 Output_.
+We will start from sample _00 intro/_05 jquery_.
 
 Summary steps:
  - Install jquery via npm.
- - Setup a global alias ($).
- - Create some sample code using this library.
- - Break into two bundles `app.js` and `vendor.js`.  
+ - Steps to buid it:
+  - Prerequisites
+  - Steps
+  - Running and testing it
```

Comment:
In the steps we summarize the detailed steps, so a developer can give  a read and check what he is going to do (e.g. you have to install this pacakge, setup this global variable and break into two bundles).